### ISSUE TITLE
Bad result of ENet-SAD on bdd dataset

### ISSUE THREAD

**Issue by britney-f** [cardwing/Codes-for-Lane-Detection#137]
Hello, 
I test your pretrained model on BDD100K dataset, but the evaluation result is not as good as the results in your paper, why? I use computeAccuracy function in train.lua to calculate the Accuracy and IOU values.
If you use another function to calculate Accuracy and IOU values, do you mind releasing it? 
Thank you!

**Comment by cardwing** [comment]
`sh ./experiments/test_ENet.sh` can yield the same performance as the reported value in my local server. Please have a look at the codes. Besides, can you show me the exact performance you reproduced?

**Comment by britney-f** [comment]
> `sh ./experiments/test_ENet.sh` can yield the same performance as the reported value in my local server. Please have a look at the codes. Besides, can you show me the exact performance you reproduced?

The Accuracy and IOU value I got in my local server are 24.138 and 12.192, respectively.

**Comment by cardwing** [comment]
This is my testing result. 
[train_ENet_new.log](https://github.com/cardwing/Codes-for-Lane-Detection/files/3528386/train_ENet_new.log)

**Comment by britney-f** [comment]
> This is my testing result.
> [train_ENet_new.log](https://github.com/cardwing/Codes-for-Lane-Detection/files/3528386/train_ENet_new.log)

Thank you!
But I still cannot figure out where is the problem, and I have not changed any code. Thank you anyway, I will look at the code carefully.

**Comment by cardwing** [comment]
I may upload the wrong model. I just test the trained model and it indeed yields the bad performance.

**Comment by britney-f** [comment]
> I may upload the wrong model. I just test the trained model and it indeed yields the bad performance.

So, can you upload your best model? Thank you!

**Comment by cardwing** [comment]
Sorry that the trained model is missing. You can just train the model and check if you can obtain similar performance. I will also train one and upload the trained model soon.

**Comment by britney-f** [comment]
> Sorry that the trained model is missing. You can just train the model and check if you can obtain similar performance. I will also train one and upload the trained model soon.

Ok, and I have another question. 
The input image size is 640* 360 during the training, and there are fc layers in your model. So, to test 1280*720  images, what should I do? Finetuning model under 1280*720 (not enough GPU memory) or directly testing 640* 360 images ?

**Comment by cardwing** [comment]
Just remove that FC layer since it does not contribute to the output. Then, you can test input images with arbitrary sizes.

**Comment by britney-f** [comment]
> Just remove that FC layer since it does not contribute to the output. Then, you can test input images with arbitrary sizes.

Got it thank you. You are a really nice guy.

**Comment by cardwing** [comment]
This is the best result I have obtained. You can just train the model from the pre-trained network.

`[9][834/834] Err1 0.04380 (0.04370) Err2 0.00000 (0.00000) Acc 43.90 (37.318) mRec 43.90 (37.318) mIOU 17.86 (16.160)`
